pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -171,6 +172,16 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	if (kc.ShutdownGracePeriod.Duration > 0 || kc.ShutdownGracePeriodCriticalPods.Duration > 0) && !localFeatureGate.Enabled(features.GracefulNodeShutdown) {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: specifying shutdownGracePeriod or shutdownGracePeriodCriticalPods requires feature gate GracefulNodeShutdown"))
 	}
+
+	if len(kc.ResolverConfig) > 0 {
```

Comment:
Comment like:

// Verify that the ResolverConfig file exists.  This doesn't guarantee it will still exist when we later try
// to open it, but if it doesn't exist, at least we can fail early.